src/components/content/order/create/OrderSubmit.tsx

```diff
@@ -198,11 +199,15 @@ function OrderSubmit(state: OrderSubmitProps): React.JSX.Element {
                                 </Paragraph>
                             </Tooltip>
                         </Col>
-                        <Col span={4}>
-                            <ApiDoc id={state.id} styleClass={serviceOrderStyles.contentTitleApi}></ApiDoc>
+                        <Col span={6}>
```

Comment:
here the gap between the ISV name and the API documentation link is not uniform when you compare the space between ISV name and ISV contact details in previous page. We need to keep everything uniform. 

If possible, we can add service vendor contact details to this page also. It will be even better.

Reply:
fixed.